### ISSUE TITLE
How to import in Ionic-angular

### ISSUE THREAD

**Issue by arturomf** [SilverTree7622/Phaser3_GUI_inspector#2]
Hi!, i'm developing a Phaser3 game inside an Ionic app. I have installed this library using npm but i don't know how to import it in my scene.

When i add PhaserGUIAction call, my code doesn't find this object or function.

Thanks!

**Comment by SilverTree7622** [comment]
Sorry for late dude,

I tried several times but I couldn't figure it out in the installing npm way,
I recommend the way of mine before I will fix this issue

I just added PGInspector file to html head element
the full project is in this URL
https://github.com/SilverTree7622/Phaser3_GUI_inspector/tree/master/zExample/ionic

I hope this gonna be helpful

**Comment by SilverTree7622** [comment]
> Hi!, i'm developing a Phaser3 game inside an Ionic app. I have installed this library using npm but i don't know how to import it in my scene.
> 
> When i add PhaserGUIAction call, my code doesn't find this object or function.
> 
> Thanks!

Sorry for late dude,

I tried several times but I couldn't figure it out in the installing npm way,
I recommend the way of mine before I will fix this issue

I just added PGInspector file to html head element
the full project is in this URL
https://github.com/SilverTree7622/Phaser3_GUI_inspector/tree/master/zExample/ionic

I hope this gonna be helpful

**Comment by arturomf** [comment]
Hi, thanks for the answer! Only for your information, I have got it working in an Ionic App installing by npm. I have added the script in angular.json file this way:

```
"scripts": [
              {
                "input": "node_modules/phaser3_gui_inspector/dist/PGInspector.js"
              }
            ]
```

And then in my Scene file, i have simply declare PhaserGUIAction this way:

`declare var PhaserGUIAction;`

After that, i have been able to use PhaserGUIAction(this) in my create() method.

Best Regards!
